packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -46,29 +46,29 @@ export const AWS_CDK_METADATA = new Set([
  * @see https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_website_region_endpoints
  */
 export const ROUTE_53_BUCKET_WEBSITE_ZONE_IDS: { [region: string]: string } = {
```

Comment:
Ordered map entries alphanumerically to facilitate scanning through it by a human.